scripts/build.sh

```diff
@@ -57,7 +57,7 @@ if "${do_build}"; then
     else
         echo "Info: Publish flag set to false, nothing will be published to DockerHub"
     fi
-    docker context create ${DOCKER_CONTEXT}
-    docker buildx create --use ${DOCKER_CONTEXT}
-    docker buildx build ${push_flag} --platform ${SUPPORTED_PLATFORMS} -t "${IMAGE_NAME}" -f "${root_dirpath}/Dockerfile" "${root_dirpath}"
+    docker context create "${DOCKER_CONTEXT}"
+    docker buildx create --use "${DOCKER_CONTEXT}"
+    docker buildx build ${push_flag} --platform "${SUPPORTED_PLATFORMS}" -t "${IMAGE_NAME}" -f "${root_dirpath}/Dockerfile" "${root_dirpath}"
```

Comment:
`${push_flag}` _cannot_ be between quotes here because if it's empty, it will be rendered `docker buildx build "" --platform .....` and so docker will think "" is and argument to the command.
I think though it is fine because we control what `push_flag` value can be, either `--push` or nothing.

The alternative is that I just do something less clever like:
```
if "${do_publish}"; then
    docker buildx build --push --platform .....
else
        docker buildx build --platform .....
fi
```
but the main downside is that 90% the docker command is repeated twice here.